shell/platform/fuchsia/flutter/main.cc

```diff
@@ -12,20 +12,20 @@
 #include "loop.h"
 #include "platform/utils.h"
 #include "runner.h"
+#include "runtime/dart/utils/root_node_singleton.h"
 #include "runtime/dart/utils/tempfs.h"
 
 int main(int argc, char const* argv[]) {
   std::unique_ptr<async::Loop> loop(flutter_runner::MakeObservableLoop(true));
 
   // Create our component context which is served later.
   auto context = sys::ComponentContext::Create();
-  sys::ComponentInspector inspector(context.get());
-
-  inspect::Node& root = inspector.inspector()->GetRoot();
+  dart_utils::RootNodeSingleton::Initialize(context.get());
 
   // We inject the 'vm' node into the dart vm so that it can add any inspect
   // data that it needs to the inspect tree.
-  dart::SetDartVmNode(std::make_unique<inspect::Node>(root.CreateChild("vm")));
+  dart::SetDartVmNode(std::make_unique<inspect::Node>(
```

Comment:
can you also do the same in the dart runner's main?

Reply:
done.